catalog/project/Kptfile

```diff
@@ -83,65 +83,3 @@ openAPI:
           values:
           - marker: ${PROJECT_ID}
             ref: '#/definitions/io.k8s.cli.setters.project-id'
-    io.k8s.cli.substitutions.project-id-sa-ref:
```

Comment:
Looks like we still have some substs/setters in the parent kptfile like `project-id-sa-owners-permissions`, `networking-namespace`, which are only used in the kcc-namespace package?

Reply:
I think we need to keep parent setters in both in order to not break the LZ tutorial.

Reply:
Does the LZ tutorial not use -R on all it's set commands?

Reply:
It does, but I seem to recall there being issues if the setter isn't defined in the intermediate/parent package. Can you test to verify this still works properly when setting a setter in the landing zone root?

Reply:
> I seem to recall there being issues if the setter isn't defined in the intermediate/parent package.

The issue with intermediate package setters in 0.39 is when doing a `kpt pkg get` it automatically inherits parent package setters with the same name. This makes for less to configure when expanding an existing setup.

For example, if you checked out a new project inside an existing LZ, then it would normally auto-inherit shared setter values from the parent so that you only need to set the ones unique to the project package.

Substitutions are not inherited tho. So they don't need to be propagated downwards.

I don't know if this issue still exists in kpt v1.

Reply:
You're right, I was mis-remembering the kpt 0.39 behavior. I have tested and this seems to be working as intended.